docs/configdoc.md

```diff
@@ -0,0 +1,117 @@
+## [kf-repo]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|url| `"http://localhost:8000/"`|Default Repository to Download eBPF Programs|
+
+## [web]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|metrics-addr|`"0.0.0.0:8898"`|Prometheus endpoint for pull/scrape the metrics|
+|kf-poll-interval|`"30s"`|Periodic Interval to scrape metrics using Prometheus|
+|n-metric-samples|`"20"`|Number of Metric Samples|
+
+## [admind]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|host|`""`|Hostname of admin|
+|username|`""`|Username of admin|
+|api-key|`""`|Api key of admin|
+|groud-id|`"0"`|Group id of admin|
+|api-enabled|`"true"`|Boolean to check api enabled or not|
+
+## [xdp-root-program]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|name|`"xdp-root"`|Name of xdp-root program|
+|artifact|`"xdp-root.tar.gz"`|Archive of xdp-root program|
+|ingress-map-name|`"root_array"`|Ingress map name of xdp-root program|
+|command|`"xdp_root"`|Command to run xdp-root program|
+|version|`"1.01"`|Version of xdp-root program|
+|is-user-program|`"false"`|Boolean to check xdp-root is kernel space or user space program|
+
+## [tc-root-program]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|name|`"tc_root"`|Name of tc_root program|
+|artifact|`"l3af_tc_root.tar.gz"`|Archive of tc_root program|
+|ingress-map-name|`"tc_ingress_root_array"`|Ingress map name of tc_root program|
+|egress-map-name|`"tc_egress_root_array"`|Egress map name of tc_root program|
+|command|`"tc_root"`|Command to run tc_root program|
+|version|`"1.0"`|Version of tc_root program|
+|is-user-program|`"false"`|Boolean to check tc_root is kernel space or user space program|
+
+## [ebpf-chain-debug]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|addr|`"0.0.0.0:8899"`|Address of ebpf chain debuging|
+|enabled|`"true"`|Boolean to check ebpf-chain-debuging is enabled or not|
+
+## [l3af-configs]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|restapi-addr|`"localhost:53000"`| Address of l3af-configs rest api |
+
+# [l3af-config-store]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|filename|`"/etc/l3afd/l3af-config.json"`|Absolute path of persistent config file|
+
+# [mtls]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|enabled| `"true"` | Boolean to check mtls enabled or not|
+|min-tls-version|`""`| Minimum tls version allowed|
+|cert-dir|`"/etc/l3af/certs"`|Absolute path of ca certificates |
```

Comment:
Presumably certificates are always stored as files in the filesystem, yes?   Note that is not best practice on Windows (see https://docs.microsoft.com/en-us/windows-hardware/drivers/install/certificate-stores) so is worth stating explicitly if this is a limitation of l3afd.